perun-core/src/main/java/cz/metacentrum/perun/core/api/MembersManager.java

```diff
@@ -1351,6 +1351,16 @@
 	 */
 	List<RichMember> getSponsoredMembers(PerunSession sess, Vo vo) throws PrivilegeException, VoNotExistsException;
 
+	/**
+	 * Gets list of all sponsored members of a VO.
+	 *
+	 * @param sess actor
```

Comment:
This is probably an incorrect description.

Reply:
Fixed.